src/Corporation/ui/modals/ExportModal.tsx

```diff
@@ -25,24 +25,26 @@ interface IProps {
 // Create a popup that lets the player manage exports
 export function ExportModal(props: IProps): React.ReactElement {
   const corp = useCorporation();
-  const possibleDivisions = corp.divisions.filter((division: Industry) => isRelevantMaterial(props.mat.name, division));
+  const possibleDivisions = [...corp.divisions.values()].filter((division: Industry) => {
+    return isRelevantMaterial(props.mat.name, division);
+  });
   if (possibleDivisions.length === 0) throw new Error("Export popup created with no divisions.");
   const defaultDivision = possibleDivisions[0];
   if (Object.keys(defaultDivision.warehouses).length === 0)
     throw new Error("Export popup created in a division with no warehouses.");
-  const [industry, setIndustry] = useState<string>(defaultDivision.name);
+  const [division, setDivision] = useState(defaultDivision);
```

Comment:
Maybe call this currentDivision? There's too many other places where "division" is wanted.